FLEDGE.md

```diff
@@ -396,8 +413,8 @@ reportWin(auctionSignals, perBuyerSignals, sellerSignals, browserSignals) {
 The arguments to this function are:
 
 *   auctionSignals and perBuyerSignals: As in the call to `generateBid()` for the winning interest group.
-*   sellerSignals: The output of `reportResult()` above, giving the seller an opportunity to pass information to the buyer.
-*   browserSignals: Similar to the argument to `reportResult()` above, though without the seller's desirability score, but with additional `interestGroupName` and `seller` fields.  This could also include some buyer-specific signal like the second-highest bid from that particular buyer.
+*   sellerSignals: The output of `reportResult()` above, giving the seller an opportunity to pass information to the buyer. In the case the winning buyer won a component auction and then went on to win the top-level auction, this is the output of component auction's seller's `reportResult()` method.
```

Comment:
nit: `In the case the` -> `In the case where`

Reply:
Done